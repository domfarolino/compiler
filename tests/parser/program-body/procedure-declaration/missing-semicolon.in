program dominic is
  procedure myFunc(string dominic in)
  begin
  end procedure
begin
end program .
