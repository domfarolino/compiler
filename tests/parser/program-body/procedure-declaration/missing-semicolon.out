[1;31mLine 5: Expected ';' after declaration, but got: 'begin'
Line 5: Error parsing program body
[0m